src/components/Header/AppHeader.tsx

```diff
@@ -7,38 +7,93 @@ const activeStyle: React.CSSProperties = {
   fontWeight: "bold",
 };
 
-const AppHeader = (): JSX.Element => {
-  const [menuOpen, setMenuOpen] = useState(false);
+const UserAvatar = () => {
+  return (
+    <div className="flex-1 navbar justify-center">
+      <div className="flex-none gap-2">
+        <button tabIndex={0} className="dropdown dropdown-end" type="button">
+          <div className="btn btn-ghost btn-circle avatar">
+            <div className="w-10 rounded-full">
+              <img
+                src="https://placeimg.com/80/80/people"
+                alt="profile avatar"
+              />
+            </div>
+          </div>
+          <ul className="mt-3 p-2 shadow menu menu-compact dropdown-content bg-base-100 rounded-box w-52">
+            <li>
+              <a href="#0">Kontrollpanel</a>
+            </li>
+            <li>
+              <a href="#0">Min side</a>
+            </li>
+            <li>
+              <a href="#0">Profil</a>
+            </li>
+            <li>
+              <a href="#0">Mine utlegg</a>
+            </li>
+            <li className="text-red-500">
+              <a href="#0">Logg ut</a>
+            </li>
+          </ul>
+        </button>
+      </div>
+    </div>
+  );
+};
 
-  const linkElements = routes.map(
-    (route) => (
-      <NavLink
-        key={route.route}
-        // className="mt-4 vektor-font font-semibold p-1"
-        className="header-link mobile-link"
-        to={route.route}
-        style={({ isActive }) => (isActive ? activeStyle : {})}
+const LoginButtons = () => {
+  return (
+    <div className="flex space-x-4 w-1/5 justify-start">
```

Comment:
Move `flex w-1/5 justify-start` to a wrapping `div`
```suggestion
    <div className="space-x-4">
```